src/local-fs.ts

```diff
@@ -72,15 +72,19 @@ export class LocalFileSystem implements FileSystem {
                             type: 'fileCreated',
                             fullPath: relPath.split(path.sep).join(pathSeparator),
                             newContent: content
-                        })));
+                        }))
+                        .catch(() => {})
```

Comment:
I vote for the first one. 
If we can do something that by default logs errors but can be overridden by a custom handler, that would be great.